mbed_tools_ci_scripts/report_third_party_ip.py

```diff
@@ -51,7 +53,8 @@ def main() -> int:
     set_log_level(args.verbose)
 
     try:
-        generate_spdx_reports(Path(args.output_dir))
+        project = generate_spdx_reports(Path(args.output_dir))
```

Comment:
If we pass a callable that converts `args.output_dir` to a `pathlib.Path` to the `type` kwarg of `parser.add_argument` we wouldn't have to do the conversion here. That might be a bit cleaner? We only use this once AFAICT so it's not that important at the moment.

Reply:
ok. will do

Reply:
418326abd2b04ddfb63cea5c08fe58e7951442d4